editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -32,7 +32,7 @@ interface TailWindOption {
   categories?: string[]
 }
 
-const TailWindOptions = TailWindList.classNames.map((className, index) => ({
+const TailWindOptions = Object.keys(TailwindClasses).map((className, index) => ({
```

Comment:
do we know how much does it cost to evaluate this?

Reply:
I shall attempt to profile this.